contracts/core/modules/IssuanceOrderModule.sol

```diff
@@ -460,33 +528,58 @@ contract IssuanceOrderModule is
 
             // Required vault balances after exchange order executed
             requiredBalances[i] = tokenBalance.add(requiredAddition);
-        }
+        }  
+
+        return requiredBalances;      
+    }
+
+    /**
+     * Check exchange orders acquire correct amount of tokens. Settle accounts for taker
+     * and relayer.
+     *
+     * @param  _order                   IssuanceOrder object containing order params
+     * @param  _fillQuantity            Quantity of Set to be filled
+     * @param  _attemptedFillQuantity   Quantity of Set taker attempted to fill
+     * @param  _orderData               Bytestring encoding all exchange order data
+     */
+    function settleOrder(
+        OrderLibrary.IssuanceOrder _order,
+        uint256 _fillQuantity,
+        uint256 _attemptedFillQuantity,
```

Comment:
It is hard to understand the delineation between `_fillQuantity` and `attemptedFillQuantity`. I don't know how we could name this better off the top of my head... but it needs to be more clear

Reply:
`_actualFillQuantity`?